packages/storybook/stories/NonReact/NonReactComposites.stories.mdx

```diff
@@ -0,0 +1,71 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+import { Canvas, Title } from '@storybook/addon-docs/blocks';
+
+<Meta id="use-composite-in-non-react-environment" title="Composites/Use Composite In Non React Environment" />
+
+# Use composites under non-react environment
+
+Although the whole composites UI part is built on top of React, you are still able to embed chat/calling composites into your html page with a simple `<script>` tag and several lines of code like:
+
+```html
+<script src="[url of chatComposite.js]"></script>
+<div id="chat" style="height: 500px"></div>
+<script>
+  const chatAdapter = await chatComposite.loadChatComposite({
+    containerId: 'chat',
+    displayName: displayName,
+    threadId: threadId,
+    endpointUrl: endpointUrl,
+    userId: user,
+    token: token
+  });
+</script>
+```
+
+Here's the step-by-step instruction to get the composite js bundle file:
```

Comment:
Is this something we plan to release through our npm package? It's not ideal to make users clone and build the repo to get some output files. Maybe for now update this to be; `Check out our sample to use a composite js bundle file in your non-react webpage:`

Reply:
I've talked with David about this before, the question is we don't want to maintain a CDN by ourselves(maybe we should)

What about we upload built js file into our github? (And it might require some build pipeline modification)

Reply:
Yep that's fair I don't want to have to maintain that 😅 Also don't fancy maintaining github releases for this either.. (disjointed if we point customers to the npm package normally but the repo for this release).
Leave this as is and we can think about it in the future, maybe it warrants its own npm package